functests/1_performance/cpu_management.go

```diff
@@ -415,6 +415,58 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", func() {
 			Expect(defaultSmpAffinitySet).To(Equal(onlineCPUsSet), fmt.Sprintf("Default SMP Affinity %s should be equal to all active CPUs %s", defaultSmpAffinitySet, onlineCPUsSet))
 		})
 	})
+
+	When("reserved CPUs specified", func() {
+		var testpod *corev1.Pod
+
+		BeforeEach(func() {
+			testpod = pods.GetTestPod()
+			testpod.Namespace = testutils.NamespaceTesting
+			testpod.Spec.NodeSelector = map[string]string{testutils.LabelHostname: workerRTNode.Name}
+
+			err := testclient.Client.Create(context.TODO(), testpod)
+			Expect(err).ToNot(HaveOccurred())
+
+			err = pods.WaitForCondition(testpod, corev1.PodReady, corev1.ConditionTrue, 10*time.Minute)
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It("should run infra containers on reserved CPUs", func() {
+			podUID := strings.Replace(string(testpod.UID), "-", "_", -1)
+
+			cmd := []string{"/bin/bash", "-c", fmt.Sprintf("find /rootfs/sys/fs/cgroup/cpuset/ -name *%s*", podUID)}
+			podCgroup, err := nodes.ExecCommandOnNode(cmd, workerRTNode)
+			Expect(err).ToNot(HaveOccurred())
+
+			cmd = []string{"/bin/bash", "-c", fmt.Sprintf("find %s -name crio-*", podCgroup)}
+			containersCgroups, err := nodes.ExecCommandOnNode(cmd, workerRTNode)
+			Expect(err).ToNot(HaveOccurred())
+
+			containerID, err := pods.GetContainerIDByName(testpod, "test")
+			Expect(err).ToNot(HaveOccurred())
+
+			containersCgroups = strings.Trim(containersCgroups, "\n")
+			containersCgroupsDirs := strings.Split(containersCgroups, "\n")
```

Comment:
What is the difference between the above 2 lines?

Reply:
The first one removes empty lines and the second one splits the rest?

Reply:
@MarSik :+1:

Reply:
This is me being tired, i missed Trim/Split...